assets/i18n/active.ko.json

```diff
@@ -167,12 +195,24 @@
     "hash": "sha1-a4fba46ee42d558010f595fec2e1e451cf3d4d73",
     "other": "**{{.Question}}** 투표가 종료되었으며, 원본 게시글이 갱신되었습니다. [여기]({{.Link}})를 눌러서 바로 이동할 수 있습니다."
   },
+  "response.resetVotes.noVotes": {
+    "hash": "sha1-18b17380e0289d021b70c7111bab1dca0d563b30",
+    "other": "초기화할 투표가 없습니다."
+  },
+  "response.resetVotes.success": {
+    "hash": "sha1-fd557010c26787d3aeece2745511dab764cb09c7",
+    "other": "모든 투표가 초기화됩니다. 이전 응답은 [{{.ClearedVotes}}] 였습니다."
+  },
   "response.vote.counted": {
     "hash": "sha1-3420b74a2fee31243e4e4e7a72a527832685f97d",
     "other": "투표 참여가 집계되었습니다."
   },
+  "response.vote.multi.updated": {
+    "hash": "sha1-213785e569b994863b96b984772e9497f8629bbc",
+    "other": "투표 참여가 집계되었습니다. {{.Remains}}개의 응답이 남았습니다."
```

Comment:
Just to double check: The translation is the same for one remaining vote and for multiple ones?

Reply:
Yes :smiley: